src/WrapCalls.cpp

```diff
@@ -0,0 +1,156 @@
+#include "WrapCalls.h"
+#include "IRVisitor.h"
+
+#include <set>
+
+namespace Halide{
+namespace Internal {
+
+using std::map;
+using std::pair;
+using std::set;
+using std::string;
+using std::vector;
+
+// Return true if 'func' exists as a copy of one of the function in 'copied_map'
+bool is_copy(const Function& func, const map<Function, Function> &copied_map) {
```

Comment:
Put helpers like this in an anonymous namespace so that they're private to this file for linking.